src/librustc_typeck/diagnostics.rs

```diff
@@ -45,8 +45,8 @@ Matching with the wrong number of fields has no sensible interpretation:
 
 ```compile_fail
```

Comment:
Note that the point of this example is to show incorrect syntax in the `match` below.